inc/alert.class.php

```diff
@@ -416,7 +416,7 @@ public function showForm($ID, $options = []) {
 
    static function displayOnCentral() {
       echo "<tr><th colspan='2'>";
-      self::displayAlerts(['show_only_login_alerts' => false]);
+      self::displayAlerts(['show_helpdesk_alerts' => true]);
```

Comment:
Maybe the parameter should be renamed to `show_only_helpdesk_alerts` to be more understandable.